doc/ref_model/chapters/chapter04.md

```diff
@@ -600,7 +600,28 @@ These flavors, similarly to other flavours, should be supported for three CNTT r
 <a name="4.3"></a>
 ## 4.3 Networking
 
-This is a placeholder for Cloud Infrastructure infrastructure networking information that is common to all Reference Architectures.
+The Cloud Infrastructure Networking Reference Model is an essential foundation that governance all Reference Architectures and Cloud Infrastructure implementations to enable a continuous evolution and migration from current single Infrastructure as a Service (IaaS) based virtualization instances with Virtual Machines (VM) into tomorrow’s multitude of Cloud Native Container as a Service (CaaS) based virtualization instances.  
```

Comment:
does the networking reference model not support the "current Infrastructure as a Service (IaaS) based virtualization instances"? I think you want to state is that it supports the "current widely used single Infrastructure as a Service (IaaS) based virtualization instances", and the evolving "multi IaaS based virtualization instances", "Cloud Native Container as a Service (CaaS) based virtualization instances" and the "hybrid multi IaaS and CaaS based virtualization instances." since that will make a very complex sentence, it may be best to insert the choices as a list.

```suggestion
The Cloud Infrastructure Networking Reference Model is an essential foundation that governs all Reference Architectures and cloud infrastructure implementations to enable multiple cloud infrastructure virtualisation technology choices and their evolution. These include:
- the current single Infrastructure as a Service (IaaS) based virtualisation instances with Virtual Machines (VM)
- multi IaaS based virtualisation instances
- Cloud Native Container as a Service (CaaS) based virtualisation instances, and
- hybrid multi IaaS and CaaS based virtualisation instances
```

Reply:
@pgoyal01 Implemented table and rewording as a more descriptive way of handling current and future networking needs, but I kept the capitalized Cloud Infrastructure since it is our specific CNTT Cloud Infrastructure instances that I talk about and not any generic cloud infrastructure instances.